generator/booking_quality_check/booking_quality_check.cpp

```diff
@@ -57,9 +60,7 @@ int main(int argc, char * argv[])
   vector<size_t> elementIndexes(elements.size());
   iota(elementIndexes.begin(), elementIndexes.end(), 0);
 
-  // In first implementation, we used random_shufle for reference dataset.
-  // Next time we are going to replace random_shuffle by shuffle with defined seed.
-  random_shuffle(elementIndexes.begin(), elementIndexes.end());
+  shuffle(elementIndexes.begin(), elementIndexes.end(), minstd_rand(FLAGS_random_seed));
```

Comment:
If you need to do a selection of k from n, you can do it much better, with O(k) memory.